pages/api/program/index.ts

```diff
@@ -0,0 +1,64 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { ResponseUtil } from "@utils/responseUtil";
+import { getPrograms, createProgram } from "@database/program";
+import { Program } from "models/Program";
+import { validateProgram } from "@utils/validation/program";
+
+/**
+ * handle controls the request made to the program resource
+ * @param req API request object
+ * @param res API response object
+ */
+export default async function handle(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    switch (req.method) {
+        case "GET": {
+            const programs = await getPrograms();
+            ResponseUtil.returnOK(res, programs);
+            res.status(200).json({});
+            break;
+        }
+        case "POST": {
+            // TODO:
+            if (!validateProgram(req.body as Program)) {
```

Comment:
I think in the future, we might have more fields in `req.body` so that it won't correspond directly to `Program`. I think a better name perhaps would be `CreateProgramInput` or `ProgramCreateInput` as it more accurately describes that the type is for creating a program.